src/store/modules/ModFilterModule.ts

```diff
@@ -54,6 +66,38 @@ export default {
             state.allowNsfw = value;
         },
 
+        setfilterDateCreatedFrom: function(state: State, value: string | null) {
+            if (value) {
```

Comment:
Would this be nicer?
`state.filterDateCreatedFrom = value ? new Date(value) : null;`